app/guid-file/template.hbs

```diff
@@ -196,32 +196,24 @@
                                     </div>
                                 {{/unless}}
                                 <dl>
-                                    <dt>{{t 'file_detail.title'}}</dt>
+                                    <dt data-test-target-title-label>{{t 'file_detail.title'}}</dt>
                                     <dd data-test-target-title>{{manager.target.title}}</dd>
-                                    <dt>{{t 'file_detail.description'}}</dt>
+                                    <dt data-test-target-description-label>{{t 'file_detail.description'}}</dt>
                                     <dd data-test-target-description>{{manager.target.description}}</dd>
-                                    {{#unless manager.isAnonymous}}
```

Comment:
This `unless` needs to be added back in

Reply:
Ok, this has been added. It had not been on upstream/guid-file/template.hbs.